src/Examples/UnitModels/Operations/feed_testing.ipynb

```diff
@@ -0,0 +1,376 @@
+      "Dual infeasibility......:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Constraint violation....:   1.4210854715202004e-14    1.4210854715202004e-14\n",
+      "Complementarity.........:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Overall NLP error.......:   1.4210854715202004e-14    1.4210854715202004e-14\n",
+      "\n",
+      "\n",
+      "Number of objective function evaluations             = 1\n",
+      "Number of objective gradient evaluations             = 1\n",
+      "Number of equality constraint evaluations            = 1\n",
+      "Number of inequality constraint evaluations          = 0\n",
+      "Number of equality constraint Jacobian evaluations   = 1\n",
+      "Number of inequality constraint Jacobian evaluations = 0\n",
+      "Number of Lagrangian Hessian evaluations             = 0\n",
+      "Total CPU secs in IPOPT (w/o function evaluations)   =      0.001\n",
+      "Total CPU secs in NLP function evaluations           =      0.000\n",
+      "\n",
+      "EXIT: Optimal Solution Found.\n",
+      "\b\b\b\b\b\b\b\b\b\b\b\b\b\b"
+     ]
+    }
+   ],
+   "source": [
+    "# Solve the simulation using ipopt\n",
+    "# Note: If the degrees of freedom = 0, we have a square problem\n",
+    "opt = SolverFactory('ipopt')\n",
+    "result = opt.solve(m, tee=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "tags": [
+     "testing",
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "from pyomo.opt import TerminationCondition, SolverStatus\n",
+    "\n",
+    "# Check if termination condition is optimal\n",
+    "assert result.solver.termination_condition == TerminationCondition.optimal\n",
+    "assert result.solver.status == SolverStatus.ok"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### View Results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "====================================================================================\n",
+      "Unit : fs.feed                                                             Time: 0.0\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Stream Table\n",
+      "                               Units        Outlet  \n",
+      "    flow_mol                mole / second     100.00\n",
+      "    mole_frac_comp benzene  dimensionless    0.60000\n",
+      "    mole_frac_comp toluene  dimensionless    0.40000\n",
+      "    temperature                    kelvin     298.15\n",
+      "    pressure                       pascal 1.0132e+05\n",
+      "====================================================================================\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Display output report\n",
```

Comment:
A text box before this would be good that says that, as expected, the outlet conditions match the feed conditions we set earlier.